src/main/kotlin/ch/bfh/habits/controllers/HabitController.kt

```diff
@@ -9,6 +9,7 @@ import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.*
 
 @RestController
+@CrossOrigin(origins = ["http://localhost:3000"])
```

Comment:
Thanks. Will configure it later for the whole app.

Reply:
k